### PR TITLE
Fix typos in ompi/ subdirectory

### PR DESCRIPTION
**Pull request by luzpaz**
Found via: `codespell -q 3 -S ./3rd-party,./docs/news -L bloc,chello,childs,connexion,fo,improbe,inout,iput,mmaped,msdos,ro,scoll,sentinal,upto`

Signed-off-by: luz paz <luzpaz@github.com>

### CODE REVIEW

File: ompi/peruse/peruse.h [hunk 1]
```diff
@@ -36,7 +36,7 @@ typedef int (peruse_comm_callback_f)(peruse_event_h event_h,
 
 /* PERUSE constants */
 enum {
-    PERUSE_SUCCESS = 0,       /* Success *//* XXX Devation from 1.11 */
+    PERUSE_SUCCESS = 0,       /* Success *//* XXX Deviiation from 1.11 */
```
Comment (jsquyres) [inline ompi/peruse/peruse.h:39]: Is this right?  Shouldn't it be "Deviation" (one "i", not two)?

Comment (luzpaz) [reply ompi/peruse/peruse.h]: Apologies, not sure how that snuck in. Fixed in [aa5a3d3](https://github.com/open-mpi/ompi/pull/10607/commits/aa5a3d36a665dae4a213b7094448e694d85ff4e0)

**Comment by jsquyres** [comment]
Awesome -- thanks!  Can you squash those 2 commits together?

**Comment by jsquyres** [comment]
> Sure, shall i squash and 'force push'?

Yes please; thanks!

**Review by jsquyres — APPROVED** [review]
You have unlocked the Level 2 spelling badge:

Lighting_spelling-1

**Comment by jsquyres** [comment]
ok to test

**Pull request merged**